app/models/configuration_script.rb

```diff
@@ -7,5 +7,14 @@ class ConfigurationScript < ApplicationRecord
   belongs_to :inventory_root_group, :class_name => "EmsFolder"
   belongs_to :manager,              :class_name => "ExtManagementSystem"
 
+  # As a job template, I have a playbook
+  has_one    :configuration_script
```

Comment:
This should be `has_many`.  Also, we may want to use the terms parent / children, but lets discuss before changing again.